src/main/java/org/graylog/integrations/notifications/types/SlackMessage.java

```diff
@@ -73,6 +76,7 @@ public String getJsonString() {
 		params.put("channel", channel);
 		params.put("text", message);
 		params.put("link_names", linkNames);
+		params.put("backlog_size",backlogSize);
```

Comment:
Why do you add the `backlog_size` to the payload that we send to Slack? Is this an officially supported field?

Reply:
we are overriding the EventNotificationSettings.FIELD_BACKLOG_SIZE using this field backlog_size and yes its an officially supported field in the graylog core code hence the override.

Reply:
Good point, we don't need this.